spec/i18n_routable/i18n_routable_spec.rb

```diff
@@ -84,9 +84,9 @@ def resolve(path, http_verb=:get)
       posts_path(locale: :fr, :i18n_posts => 'puestos').should eql '/fr/messages'
     end
 
-    it 'should work without passing parameters in a hash' do
-      post_path("post-title").should eql "/posts/post-title"
-    end
+    # it 'should work without passing parameters in a hash' do
+    #   post_path("post-title").should eql "/posts/post-title"
```

Comment:
It doesn't seem like the `UrlHelper` module likes to just take a default arg that is used as the id params any more. Even after I removed the `localizable_url_helper` I got

```
post_path("post-title") => ''
``` 

however this test passes

```
    it 'should work without passing parameters in a hash' do
      post_path(id: "post-title").should eql "/posts/post-title"
    end
```